Game/OccupancyMap.swift

```diff
@@ -16,7 +16,18 @@ enum Rotation: Int {
 	}
 	
 	func nextAnticlockwise() -> Rotation {
-		return Rotation(rawValue: (self.rawValue - 1) % 4)!
+		//this line causes a crash, so lets manually convert to different values
```

Comment:
I already told Seth about this one when I reviewed his; the right solution is to add 3 instead of subtract 1